alpine-php/builder/Dockerfile.tmpl

```diff
@@ -56,6 +57,7 @@ RUN npm install --global npm@latest && \
       gzip \
       openssh-client \
       patch \
+      python3 \
```

Comment:
Cool story: HPC Viewer theme asset generation hard requires python2.

Reply:
warning, from alpine 3.12, the python 2 package has been renamed from `python` to `python2`

Reply:
It can go and update its workflow. We don't support python2.